### ISSUE TITLE
[Python] Broken highlighting of multi-line with statements

### ISSUE THREAD

**Issue by philer** [sublimehq/Packages#3674]
### What happened?

Since Python 3.10, `with` statements with multiple values can be broken into multiple lines with parenthesis.

Example from the [official documentation](https://docs.python.org/3/reference/compound_stmts.html#the-with-statement):

```python
with (
    A() as a,
    B() as b,
):
    SUITE
```

Reproduced as valid code:
```python
from contextlib import contextmanager

@contextmanager
def A():
    yield "foo"

@contextmanager
def B():
    yield "foo"


with (
    A() as a,
    B() as b,
):
    ...
```
Looks like this in my editor (Build 4143):
![broken-with-statement-highlighting](https://user-images.githubusercontent.com/4589347/224366082-75577cbe-e3b3-4a6b-a020-eef9f50358c8.png)

I have a bunch of plugins installed but to the best of my knowledge none should interfere with Python syntax highlighting.

Note: There was a related issue (#3619) which was handled in #3620.

**Comment by deathaxe** [comment]
This issue is fixed by #3620. The PR is shipped with ST4144+.

**Comment by jfcherng** [comment]
Pretty sure the given reproducer has been fixed on the current master.
